deploy/webhook/cloudbuild.yaml

```diff
@@ -0,0 +1,41 @@
+steps:
+  # Get cluster credentials
+- name: gcr.io/cloud-builders/gcloud
```

Comment:
you could simplify this by using skaffold :)

Reply:
```
steps:
- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
  args: ['skaffold', 'run', '-f=skaffold.yaml']
```

Reply:
thanks Matt! good call :)